vnext/ReactCommon/TEMP_UntilReactCommonUpdate/react/renderer/components/view/AccessibilityProps.cpp

```diff
@@ -0,0 +1,261 @@
+                    context,
+                    rawProps,
+                    "onAccessibilityTap",
+                    sourceProps.onAccessibilityTap,
+                    {})),
+      onAccessibilityMagicTap(
+          CoreFeatures::enablePropIteratorSetter
+              ? sourceProps.onAccessibilityMagicTap
+              : convertRawProp(
+                    context,
+                    rawProps,
+                    "onAccessibilityMagicTap",
+                    sourceProps.onAccessibilityMagicTap,
+                    {})),
+      onAccessibilityEscape(
+          CoreFeatures::enablePropIteratorSetter
+              ? sourceProps.onAccessibilityEscape
+              : convertRawProp(
+                    context,
+                    rawProps,
+                    "onAccessibilityEscape",
+                    sourceProps.onAccessibilityEscape,
+                    {})),
+      onAccessibilityAction(
+          CoreFeatures::enablePropIteratorSetter
+              ? sourceProps.onAccessibilityAction
+              : convertRawProp(
+                    context,
+                    rawProps,
+                    "onAccessibilityAction",
+                    sourceProps.onAccessibilityAction,
+                    {})),
+      importantForAccessibility(
+          CoreFeatures::enablePropIteratorSetter
+              ? sourceProps.importantForAccessibility
+              : convertRawProp(
+                    context,
+                    rawProps,
+                    "importantForAccessibility",
+                    sourceProps.importantForAccessibility,
+                    ImportantForAccessibility::Auto)),
+      testId(
+          CoreFeatures::enablePropIteratorSetter ? sourceProps.testId
+                                                 : convertRawProp(
+                                                       context,
+                                                       rawProps,
+                                                       "testID",
+                                                       sourceProps.testId,
+                                                       "")) {
+  // It is a (severe!) perf deoptimization to request props out-of-order.
+  // Thus, since we need to request the same prop twice here
+  // (accessibilityRole) we "must" do them subsequently here to prevent
+  // a regression. It is reasonable to ask if the `at` function can be improved;
+  // it probably can, but this is a fairly rare edge-case that (1) is easy-ish
+  // to work around here, and (2) would require very careful work to address
+  // this case and not regress the more common cases.
+  if (!CoreFeatures::enablePropIteratorSetter) {
+    const auto *rawPropValue =
+        rawProps.at("accessibilityRole", nullptr, nullptr);
+    AccessibilityTraits traits;
+    std::string roleString;
+    if (rawPropValue == nullptr || !rawPropValue->hasValue()) {
+#ifdef USE_WINUI_FABRIC
+      traits = sourceProps.accessibilityTraits;
```

Comment:
Is this the fix you're trying to get? We already consume a new enough RN version to have this code.

Reply:
Yea my bad! We didn't have this code so I thought Microsoft might not have it as well. I'll close up this PR